moz_ingest/io_modules/decoders/moz_ingest/common.lua

```diff
@@ -291,7 +295,7 @@ function transform_message(hsr, msg)
         if not ip_addr then
             ip_addr = hsr:read_message("Fields[remote_addr]")
         end
-        msg.Fields.geoCity, msg.Fields.geoCountry = get_geo_city(ip_addr)
+        msg.Fields.geoCity, msg.Fields.geoCityGeonameId, msg.Fields.geoCountry = get_geo_city(ip_addr)
```

Comment:
@trink Do we need to explicitly set the `representation` for the `geoCityGeonameId` field to integer if we don't want it to show up as a float?

Reply:
The 'value_type' would have to be set to 2 otherwise the numeric value is defaulted to a double.